query/src/storages/fuse/operations/commit.rs

```diff
@@ -109,12 +158,12 @@ impl FuseTable {
     }
 
     async fn commit_to_meta_server(
-        &self,
+        tbl_id: &TableIdent,
```

Comment:
How about re-order the arguments to:
```
  ctx: &QueryContext,
  tbl_id: &TableIdent,
```
?

Reply:
:ok_hand: